packages/tracing/src/browser/metrics/index.ts

```diff
@@ -34,16 +34,18 @@ let _clsEntry: LayoutShift | undefined;
 /**
  * Start tracking web vitals
  */
-export function startTrackingWebVitals(): void {
+export function startTrackingWebVitals(): ReturnType<typeof _trackLCP> {
   const performance = getBrowserPerformanceAPI();
   if (performance && browserPerformanceTimeOrigin) {
     if (performance.mark) {
       WINDOW.performance.mark('sentry-tracing-init');
     }
     _trackCLS();
-    _trackLCP();
     _trackFID();
+    return _trackLCP();
```

Comment:
Does CLS also have a stop listen? we should be returning all of them in a wrapper function and just settling them whenever we finish the transaction

Reply:
yeah that makes sense to me, will make the change.